.gitignore

```diff
@@ -47,3 +47,4 @@ bower.json
 # Temporary Paperclip test files.
 /public/test_files
 .zedstate
+.DS_Store
```

Comment:
You might want to add .DS_Store to [your global ~/.gitignore file](https://help.github.com/articles/ignoring-files/#create-a-global-gitignore) so that you don't need to do it in every project.